src/main/java/teammates/storage/sqlentity/FeedbackResponseComment.java

```diff
@@ -111,6 +111,10 @@ public FeedbackResponse getFeedbackResponse() {
         return feedbackResponse;
     }
 
+    public FeedbackQuestion getFeedbackQuestion() {
+        return getFeedbackResponse().getFeedbackQuestion();
```

Comment:
can we just access getFeedbackResponse().getFeedbackQuestion() instead of creating a getter for this here? 

the rationale is that FeedbackResponseCommnet should not be responsible for choosing the fields of the member classes should expose, the FeedbackResponse class itself should be the one deciding since we are already providing getFeedbackResponse() in FeedbackResponseComment already.